pages/learn/index.vue

```diff
@@ -52,14 +52,12 @@ export default class LearnPage extends QiskitPage {
       }
     },
     {
-      title: SOCIAL_MEDIA.stack.label,
-      description: `Have a question? Ask it on stack exchange! Qiskit
-      advocates and core contributors monitor this forum and will happily
-      answer your questions.`,
+      title: SOCIAL_MEDIA.support.label,
+      description: `If you need support on anything relating to using and developing Qiskit, you can use the following forums to direct your issue/queries to the appropriate audience.`,
       cta: {
-        url: SOCIAL_MEDIA.stack.url,
-        label: 'Visit Stack Exchange',
-        segment: { cta: 'stack-exchange', location: 'helpful-resources' }
+        url: SOCIAL_MEDIA.support.url,
+        label: 'Visit Support Channels',
+        segment: { cta: 'support-channels', location: 'helpful-resources' }
```

Comment:
I am not sure the consequence of renaming the cta. Does it affect historical analytics data?

Reply:
I think this is okay - as an internal team, we understand this change and now have a historical _marker_ (this PR), in case we ever need to collect data on this particular CTA...

@eddybrando - what do you think?